tests/jax/components/executing/base_test.py

```diff
@@ -0,0 +1,113 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing ExecutorInit class for system builders"""
+
+from types import SimpleNamespace
+
+import pytest
+
+from mava.components.jax.executing.base import ExecutorInit, ExecutorInitConfig
+from mava.systems.jax.builder import Builder
+from mava.systems.jax.executor import Executor
+
+
+@pytest.fixture
+def dummy_config() -> ExecutorInitConfig:
+    """Dummy config attribute for ExecutorInit class
+
+    Returns:
+        ExecutorInitConfig
+    """
+    return ExecutorInitConfig(interval={"test": 1})
+
+
+def network_factory():
+    """Function used in builder.store.networ_factory"""
+    return "after_network_factory"
+
+
+@pytest.fixture
+def mock_builder() -> Builder:
+    """Mock builder component.
+
+    Returns:
+        Builder
+    """
+    builder = Builder(components=[])
+    # store
+    store = SimpleNamespace(network_factory=network_factory, networks=None)
+    builder.store = store
+    return builder
+
+
+@pytest.fixture
+def mock_executor() -> Executor:
+    """Mock executor component.
+
+    Returns:
+        Executor
+    """
+    store = SimpleNamespace(is_evaluator=None, observations={})
+    executor = Executor(store=store)
+    executor._interval = None
+    return executor
+
+
+# test on_building_init
```

Comment:
Also, please remove these comments above each test method as I do not think that they are needed